app/build.gradle

```diff
@@ -19,11 +19,11 @@ android {
     compileSdkVersion versions.compileSdk
     buildToolsVersion versions.buildTools
     defaultConfig {
-        applicationId "org.fossasia.eventyay"
+        applicationId "com.eventyay.organizer"
         minSdkVersion versions.minSdk
         targetSdkVersion versions.targetSdk
-        versionCode 4
-        versionName "1.0.3alpha"
+        versionCode 5
+        versionName "1.0.0beta"
```

Comment:
Should be 1.0.4alpha

Reply:
Ok, I've changed it to alpha but Mario told in the gitter chat that we need to release beta version now. Aren't we releasing beta ?

Reply:
We'll release in the beta channel on Play Store, the app itself is in alpha phase

Each release of app on Play Store goes through alpha -> beta -> release

Even Whatsapp which is not in alpha or beta, Play Store channels are unrelated to app state